metadata/metadata_service/api/feature.py

```diff
@@ -0,0 +1,131 @@
+from typing import Any, Iterable, Mapping, Union
+
+from flasgger import swag_from
+from flask_restful import Resource, reqparse
+
+from metadata_service.api.badge import BadgeCommon
+from metadata_service.api.tag import TagCommon
+from metadata_service.proxy import get_proxy_client
+
+
+class FeatureDetailAPI(Resource):
+    """
+    FeatureDetail API
+    """
+
+    def __init__(self) -> None:
+        self.client = get_proxy_client()
```

Comment:
Is it worth having a base resource to reduce the copy/paste of the `init`s in this file? For example:

```python
class MyResource(Resource):
    def __init__(self) -> None:
        self.client = get_proxy_client()
...
class FeatureDetailAPI(MyResource):
...
```

Reply:
This is a good idea, I wonder if we should take that abstraction even higher up so Tables and other entities can reuse it as well. Might be a bit out fo the scope of this PR to make the change for all entities.

Reply:
I think we should do this as a separate PR @sewardgw so we can apply it to all entities, what do you think?

Reply:
👍🏻  sounds good!